File: 10kali.py
#program menerima input bilangan dari user sebanyak 10 kali dan menampilkan bilangan terkecil
import sys

def angka_terkecil():
    angka = 1
    angka_terkecil = None
    print("Masukkan 10 angka:")
    for i in range(10):
        input_string = input(f"Angka ke-{i + 1}: ")
        try:
            angka = int(input_string)
        except ValueError:
            print("Input tidak valid. Program berhenti.")
            return
        if i == 0:
            angka_terkecil = angka
            continue 
        for j in range(angka + 1):
            if angka_terkecil is not None and angka_terkecil > angka:
                angka_terkecil = angka
    print(f"\nAngka terkecil yang dimasukkan adalah: {angka_terkecil}")
    
angka_terkecil()

File: 10kali.c
#include <stdio.h>
//program menerima input bilangan dari user sebanyak 10 kali dan menampilkan bilangan terkecil
int main()
{
    int angka;
    int angka_terkecil;
    int i,j;

    printf("Masukkan 10 angka:\n ");
    for (i=0; i<10; i++){
        scanf("%d", &angka);
        for (j=0; j<=angka; j++){
            if (angka_terkecil>angka){
                angka_terkecil=angka;
            } j++;
        }
    }
    printf("Angka terkecil yang dimasukkan adalah: %d",angka_terkecil);

    return 0;
}